### PR TITLE
[Merged by Bors] - style(test/linarith): adapt to more mathlib style

### PR DESCRIPTION
**Pull request by grunweg**
- put `by` on the previous line
- indent subsequent hypotheses by four spaces
- indent conclusions also by four spaces
- remove some double spaces

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: test/linarith.lean [hunk 1]
```diff
@@ -188,8 +186,8 @@ example (i n : ℕ) (h : (2:ℤ) ^ i ≤ 2 ^ n) : (0:ℤ) ≤ 2 ^ n - 2 ^ i := b
 example (a b c : Rat) (h2 : b > 0) (h3 : b < 0) : Nat.prime 10 := by
   linarith
 
-example (a b c : Rat) (h2 : (2 : Rat) > 3)  : a + b - c ≥ 3 :=
```
Comment (eric-wieser) [inline test/linarith.lean:191]: I would leave this, it doubles a test that `linarith` works on `>`

File: test/linarith.lean [hunk 2]
```diff
@@ -73,14 +73,12 @@ example (ε : Rat) (h1 : ε > 0) : ε / 2 + ε / 3 + ε / 7 < ε :=
  by linarith
 
 example (x y z : Rat) (h1 : 2*x < 3*y) (h2 : -4*x + z/2 < 0)
-        (h3 : 12*y - z < 0)  : False :=
-by linarith
+    (h3 : 12*y - z < 0)  : False := by
```
Comment (eric-wieser) [inline test/linarith.lean:76]: nit:
```suggestion
    (h3 : 12*y - z < 0) : False := by
```
(and a few others)

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

File: test/linarith.lean (hunk 1 above)
Comment (grunweg) [reply test/linarith.lean:191]: Oh, indeed! Reverted.

**Comment by grunweg** [comment]
Thank you for the fast review!
bors r+

**Pull request closed without merging**